src/com/lms/service/AdministratorService.java

```diff
@@ -27,7 +30,7 @@
 	 * @param publisher the publisher of the book
 	 * @return the newly-created book
 	 */
-	Book createBook(String title, Author author, Publisher publisher) throws SQLException;
+	Book createBook(String title, Author author, Publisher publisher) throws InsertException;
```

Comment:
Throughout, I would declare each method (in the interface) `throws TransactionException` rather than the specific subclass, just in case at some point an implementation needs to use one of the others.